<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSavePointSet</class>
 <widget class="QDialog" name="DialogSavePointSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Point Set</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter file name</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditFileName">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonOpen">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="freeview.qrc">
         <normaloff>:/resource/icons/file_open_16.png</normaloff>:/resource/icons/file_open_16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Save as</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButtonWayPoint">
       <property name="text">
        <string>Way Points</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonControlPoint">
       <property name="text">
        <string>Control Points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonEnhanced">
       <property name="text">
        <string>Enhanced Point Set (Json)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="freeview.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButtonOpen</sender>
   <signal>clicked()</signal>
   <receiver>DialogSavePointSet</receiver>
   <slot>OnOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSavePointSet</receiver>
   <slot>OnOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSavePointSet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonWayPoint</sender>
   <signal>clicked()</signal>
   <receiver>DialogSavePointSet</receiver>
   <slot>OnRadioButtonFileType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonControlPoint</sender>
   <signal>clicked()</signal>
   <receiver>DialogSavePointSet</receiver>
   <slot>OnRadioButtonFileType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonEnhanced</sender>
   <signal>clicked()</signal>
   <receiver>DialogSavePointSet</receiver>
   <slot>OnRadioButtonFileType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnOpen()</slot>
  <slot>OnOK()</slot>
  <slot>OnRadioButtonFileType()</slot>
 </slots>
</ui>
